### ISSUE TITLE
"Your organization blocked this file because it didn't meet a security policy" in Chrome

### ISSUE THREAD

**Issue by likethesky** [EFForg/privacybadger#3004]
I get "Your organization blocked this file because it didn't meet a security policy" when trying to download legitimate PDF forms from client.schwab.com ... After adding client.schwab.com to the excluded domains on Privacy Badger, the problem stopped.

**Comment by ghostwords** [comment]
Hello and thanks for opening an issue!

Is this in Chrome? To clarify, which domain is the PDF hosted on, and what is the domain of the site you are on when this happens?

**Comment by ghostwords** [comment]
This might be another Google's Manifest V3 bug where [Declarative Net Request's definition of "thirdParty"](https://developer.chrome.com/docs/extensions/reference/api/declarativeNetRequest#type-DomainType) is different from what extensions expect. Google's DNR defines the "thirdParty" filter with respect to the containing frame, while extensions like Privacy Badger expect the check to be made against the top-level document. This matters when a resource is loaded in a nested frame.

But I can't yet say for sure because I haven't yet been able to reproduce this issue.

**Comment by ckuethe** [comment]
I get the same error trying to download images from Google Voice conversations. Disabling PB on Google Voice  allows me to save images.

PB 2024.07.17, Chrome 127 & 128 from Google DEB repo, Ubuntu 22.04LTS x86_64

**Comment by ghostwords** [comment]
Haven't gotten any "Your organization blocked this file because it didn't meet a security policy" in a while. We started getting reports after the MV3 release to Chrome. The most recent report was from September 9th.

**Comment by ckuethe** [comment]
@ghostwords I can still reproduce this with Google Voice, PB 2024.7.17, Chrome 131.0.6778.85, Ubuntu 22.04LTS x86_64

**Comment by ghostwords** [comment]
@ckuethe Are you able to reproduce this issue in a [new Chrome profile](https://support.google.com/chrome/answer/2364824) with Privacy Badger?

**Comment by ckuethe** [comment]
No.  For fun, I created a brand new user on my machine and ran chrome with only PB (default settings) installed. Google Voice attachments saved correctly.

I suppose I need to see if I can repro with my main PB settings copied to my test profile, and possibly seeing which other extension might be interacting.

**Comment by Anth0n** [comment]
If your Chrome browser has not indeed been taken over by an organization, it is quite likely that some of your browser extensions are causing the issue, such as ad-blocking extensions and IDM (Internet Download Manager) extensions. You can try disabling them to resolve the problem.

**Comment by Shixe04** [comment]
I'm having the same issue here with newly setup personal pc, I was able to still download any .exe files a day prior and all of a sudden, I'm now getting the error when I tried to download legitimate .exe files that comes directly from google.

I have download a certain amount of products, but the last download that I made was for Riot Games Valorant, after the download and installation, I'm now getting the error message that **"Your Organization  blocked this file because it didn't meet a security policy"**, I have uninstalled the program, but still to no avail I'm getting the same error message.
